src/extends/room/meta.js

```diff
@@ -182,3 +182,11 @@ Room.isCuldesac = function (roomName, entrance) {
   // keeper room, or dead end appeared.
   return Object.keys(exits).length === 1
 }
+
+Room.isAvailable = function (name) {
+  if (Game.gcl.level > GCL_NOVICE) {
```

Comment:
Did you mean `>=`? You can't claim a new room in a novice area when your GCL === GCL_NOVICE.